packages/react-components/src/components/table-output-component.tsx

```diff
@@ -331,13 +301,31 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                     headerName: columnHeader.name,
                     field: columnHeader.id.toString(),
                     width: this.props.columnWidth,
-                    resizable: true
+                    resizable: true,
+                    cellRenderer: 'cellRenderer',
+                    cellRendererParams: {
+                        filterModel: this.filterModel,
+                        searchResultsColor: this.props.backgroundTheme === 'light' ? '#cccc00' : '#008000'
+                    },
+                    suppressMenu: true,
+                    filter: 'agTextColumnFilter',
+                    floatingFilter: true,
+                    floatingFilterComponent: 'searchFilterRenderer',
+                    floatingFilterComponentParams: {
+                        suppressFilterButton: true,
+                        onFilterChange: this.searchEvents,
+                        onclickNext: this.findNextMatchedEvent,
```

Comment:
I expected it to find the next matched event incrementally as I type in the text box, not only after pressing Enter or clicking the next button.

It creates this strange behavior:

1. Type 'x' where x matches in that column at row 100.
The table is updated to dim the visible rows 0-10.
I expected it to go to row 100.

2. Press Enter or click Next button.
The table goes to and selects row 100.

3. Delete 'x' so that regex is empty
The table goes back to row 0 and undims all the rows.
I expected it to stay at row 100.
(note, row 100 is still selected even though it is not visible)

4. Type 'x' again
The table is updated to dim visible rows 0-10

5. Press Enter or click Next button
The table goes to and selects the next match at row 200.
This is OK since the selection was row 100, but it's confusing because the table had scrolled back to row 0.